util/ubuntu-setup

```diff
@@ -156,6 +160,11 @@ fi
 # install all of the packages listed in required_packages.txt
 apt-get install $ARGS $(sed 's/#.*//;/^$/d' $BASE/util/ubuntu-packages.txt)
 
+if [ "$SYSTEM" = "ubuntu-14.04" ]; then
```

Comment:
Should this be == instead of =?

Also, can you put a comment above the `if` with a short description of what this is for?